components/OptionsMenu.js

```diff
@@ -0,0 +1,137 @@
+import styled from "styled-components";
+import { useState } from "react";
+import Link from "next/link";
+
+export default function OptionsMenu({ onClose, onDeleteEmotion, emotion }) {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  function toggleDeleteDialog() {
+    setIsDialogOpen(!isDialogOpen);
+  }
+
+  function handleDelete(emotion) {
```

Comment:
The emotion parameter is not used in this function. You can delete as you already have emotion from the props.
```suggestion
  function handleDelete() {
```